core/src/main/resources/defaults/blueprints/cdp-data-mart-active.bp

```diff
@@ -0,0 +1,226 @@
+{
+  "description": "CDP 1.0 Active Data Mart template with Apache Impala, Hue, and Kudu",
+  "blueprint": {
+    "cdhVersion": "7.0.0",
+    "displayName": "datamart-active",
+    "services": [
+      {
+        "refName": "hdfs",
+        "serviceType": "HDFS",
+        "roleConfigGroups": [
+          {
+            "refName": "hdfs-NAMENODE-BASE",
+            "roleType": "NAMENODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-SECONDARYNAMENODE-BASE",
+            "roleType": "SECONDARYNAMENODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-DATANODE-BASE",
+            "roleType": "DATANODE",
+            "configs" : [ {
+              "name" : "dfs_datanode_max_locked_memory",
+              "value" : "0",
+              "autoConfig" : false
+            } ],
+            "base": true
+          },
+          {
+            "refName": "hdfs-BALANCER-BASE",
+            "roleType": "BALANCER",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "kudu",
+        "serviceType": "KUDU",
+        "serviceConfigs" : [ {
+          "name" : "gflagfile_service_safety_valve",
+          "value" : "--use_hybrid_clock=false"
+        } ],
+        "roleConfigGroups": [
+          {
+            "refName": "kudu-MASTER-BASE",
+            "roleType": "KUDU_MASTER",
+            "base": true,
+            "configs" : [ {
+              "name" : "fs_wal_dir",
+              "value" : "/hadoopfs/fs1/master"
+            }, {
+              "name" : "fs_data_dirs",
+              "value" : "/hadoopfs/fs1/master"
+            } ]
+          },
+          {
+            "refName": "kudu-TSERVER-BASE",
+            "roleType": "KUDU_TSERVER",
+            "base": true,
+            "configs" : [ {
+              "name" : "fs_wal_dir",
+              "value" : "/hadoopfs/fs1/tserver"
```

Comment:
Should this be a different directory than the data dir?

Reply:
Now irrelevant given that I've removed the hardcoded directories.